skepticoin/scripts/mine.py

```diff
@@ -52,20 +58,29 @@ def main():
 
             nonce = random.randrange(1 << 32)
             last_round_second = int(time())
-            i = 0
+            hashes = 0
 
             while True:
                 if int(time()) > last_round_second:
-                    print("Hashrate:", i)
+                    now = datetime.now()
+                    now_str = now.strftime("%Y-%m-%d %H:%M:%S")
+                    uptime = now - start_time
+                    uptime_str = str(uptime).split(".")[0]
+
+                    balance = wallet.get_balance(coinstate) / Decimal(SASHIMI_PER_COIN)
```

Comment:
TBH `wallet.get_balance` is an operation that has a computational cost that is so high that doing it every second will lead to a visible decrease in hashrate. The real solution is to make it less computationally expensive; if that's not something your up for (yet) you'll have to avoid the call here instead.

Reply:
Fair enough. We can assume that the balance didn't change until we mine a block. I'll make some changes to fix this.

Reply:
In fact there are many more things wrong with just looking at the wallet balance (i.e. it assumes that there are no other flows in or out of the wallet) but for some informative text on screen none of that matters (and moreover, we're not really gonna pretend real transactions are a usecase of skepticoin...)